lib/licensed/dependency.rb

```diff
@@ -142,6 +142,7 @@ def license_metadata
     def generated_license_contents
       return unless license
       return if license.key == "other"
+      return unless license.text.nil?
```

Comment:
I think this is backwards, the function should return when the text is nil.

```suggestion
      return if license.text.nil?
```